### ISSUE TITLE
Separate image for face-down Cybermen

### ISSUE THREAD

**Issue by Glorax** [Card-Forge/forge#3959]
**Is your feature request related to a problem? Please describe.**
Currently, face-down Cybermen use the exact same image as whichever image is being used for Morph cards. For example, whichever image is being used for "morph.jpg" / "morph.png" will be used for the Cybermen.

**Describe the solution you'd like**
Cybermen should use the actual separate image that was created for them. It's a completely different mechanic that shouldn't use the Morph overlay.

**Describe alternatives you've considered**
I attempted to rename the Cyberman token image "morph_who.png" in order to differentiate it from the regular morph image, but this didn't work. The Morph image cannot have a suffix to be recognized, even though the Manifest image seems to work fine with suffixes.

**Comment by Hanmac** [comment]
@kevlahnota this part does the logic:

https://github.com/Card-Forge/forge/blob/d21c8818bda00ba3f1a10e7cf3fe615ce5d9156d/forge-game/src/main/java/forge/game/card/CardView.java#L1225-L1235

a facedown card on the battlefield without manifest is "morph down"

**Comment by kevlahnota** [comment]
Yes i know the override for tokenkey, its just that any face down uses the morph image if not manifested, is that a general rule? Or any image can be used for morph? I just want to distinguish the facedown state like the OP requested since there's no ismorph() function on the Card class unlike the ismanifested()

**Comment by Hanmac** [comment]
from the current logic, any face down that is not manifested is morph down (even if it might not have a morph up cost)

**Comment by kevlahnota** [comment]
So I guess just make a token key for the cybermen, if its facedown non manifested, if it has subtypes use cybermen token key instead? Maybe parse the subtype for generic function and look for that key incase wotc put a similar effect like this.

**Comment by Hanmac** [comment]
@kevlahnota maybe for the few corner cases where an extra Facedown image exist, we could add an extra property for this?

**Comment by kevlahnota** [comment]
ok manage to set the token key for the card however what is the rules for facedown? example a morphed creature in the battlefield targeted with cyber conversion? would that turn the facedown card faceup? seems weird enough.. I thought it will just add the cyberman characteristics to the card since it's already facedown... 

https://github.com/Card-Forge/forge/assets/9781539/0b9ef673-4e13-4121-98b5-059a2e2f38fb

**Comment by kevlahnota** [comment]
@Hanmac I think setstate is missing this and maybe others: 

> 708.2b A face-down permanent can’t be turned face-down. If a spell or ability attempts to turn a face-down permanent face down, nothing happens and that effect doesn’t change any of its characteristics or their copiable values.
